src/components/flame-graph/Canvas.js

```diff
@@ -60,157 +60,6 @@ const ROW_HEIGHT = 16;
 const TEXT_OFFSET_START = 3;
 const TEXT_OFFSET_TOP = 11;
 
-function stringToHex(hexString: string): number {
-  if (hexString.length !== 7 || hexString[0] !== '#') {
-    throw new Error('Expect a hex string of the form "#000000"');
-  }
-
-  // Go from a string of the form "#000000" to a number of the form 0x000000.
-  return parseInt(hexString.substr(1), 16);
-}
-
-// stringToHex is needed for interpolated colors
-const NATIVE_START_COLOR = stringToHex('#c1e0ff');
-const NATIVE_END_COLOR = stringToHex(colors.BLUE_50);
-const NATIVE_HOVER_COLOR = colors.BLUE_60;
-const NATIVE_WHITE_TEXT_THRESHOLD = 0.8;
-
-const JS_START_COLOR = stringToHex('#ffd79f');
-const JS_END_COLOR = stringToHex(colors.ORANGE_50);
-const JS_HOVER_COLOR = colors.ORANGE_60;
-
-const UNSYMBOLICATED_START_COLOR = stringToHex(colors.GREY_30);
-const UNSYMBOLICATED_END_COLOR = stringToHex(colors.GREY_50);
-const UNSYMBOLICATED_HOVER_COLOR = colors.GREY_60;
-const UNSYMBOLICATED_WHITE_TEXT_THRESHOLD = 0.66;
-
-/**
- * Linear interpolation.
- */
-function lerp(a: number, b: number, t: number): number {
-  return a * (1 - t) + b * t;
-}
-
-function lerpColors(hexColorA: number, hexColorB: number, t: number): string {
-  // Take the hex value, and get R G B values ranged 0-255.
-  const rA = (0xff0000 & hexColorA) >> 16;
-  const gA = (0x00ff00 & hexColorA) >> 8;
-  const bA = 0x0000ff & hexColorA;
-  const rB = (0xff0000 & hexColorB) >> 16;
-  const gB = (0x00ff00 & hexColorB) >> 8;
-  const bB = 0x0000ff & hexColorB;
-  const r = lerp(rA, rB, t);
-  const g = lerp(gA, gB, t);
-  const b = lerp(bA, bB, t);
-
-  // Turn this back into a value like 0x000000, but include an extra bit so it takes the
-  // form 0x1000000 and can keep the leading zeros.
-  const modifiedHex = 0xf000000 | (r << 16) | (g << 8) | b;
-
-  return '#' + modifiedHex.toString(16).substr(1);
-}
-
-/**
- * Return the background color used to paint a flame graph box. The
- * passed stack type determines the hue while the self time determines
- * the lightness of the color. Darker colors indicate higher self
- * times.
- */
-export function getBackgroundColor(
```

Comment:
By removing this, we no longer will see higher self-time encoded as a darker color. I thought it was neat as it made it easier to identify hot code. Don't we want that anymore?

Reply:
I'd like to move forward with the work here to consolidate the color categories, however I did file #1643 as a discussion issue around this change. I'm a bit sad with removing the work we did around coming up with this coloring scheme, but let's chat about this in that discussion issue.